src/engagement_db_to_analysis/configuration.py

```diff
@@ -37,6 +37,7 @@ class AnalysisDatasetConfiguration:
     raw_dataset: str
     dataset_name: str
     coding_configs: [CodingConfiguration]
+    rapid_pro_non_relevant_label: str = None
```

Comment:
I think we should add as a todo somewhere that this eventually needs to move to the db -> rapid pro sync configuration